BioCro/DARPA/set.MA.trt.R

```diff
@@ -0,0 +1,90 @@
+## Function that creates a link table between treatments and their RE indices 
+## And produces a list of mcmc.list objects for only the treatment-level posteriors
+## Target site/treatment is re-labeled "beta.o" so additional PEcAn functions will work 
+
+
+set.MA.trt <- function(settings){
+  
+  library(coda)
+  
+  #load jagged.data and trait.mcmc
+  load(paste0(settings$outdir, "pft/SetariaWT_ME034/jagged.data.Rdata"))
```

Comment:
```suggestion
  # this will only work for a single PFT run e.g. will break w/ ED2 but ok for now
  load(file.path(settings$outdir, "pft", settings$pft$name, "jagged.data.Rdata"))
```